velox/connectors/hive/SplitReader.cpp

```diff
@@ -157,6 +232,7 @@ void SplitReader::prepareSplit(
       throw;
```

Comment:
Does throw the current velox runtime exception: e? Thanks!

Reply:
It rethrows the current exception object, `e`.
Thanks